models/clm/tutorial/README.rst

```diff
@@ -0,0 +1,1591 @@
+These should have been compiled during Step 11 of this tutorial.
+The important DART executables for the diagnostics are
+``obs_diag`` and ``obs_seq_to_netcdf``.  If they do not exist,
+perform the ``./quickbuild.csh -mpi`` command to create them.
+
+Next generate a text file that includes all the ``obs_seq*.final``
+files from the tutorial simulation
+
+::
+ 
+ > ls <rundir>/*final > obs_seq_files_tutorial.txt 
+
+Next edit the ``&obs_diag_nml`` namelist within the ``input.nml``.
+to assign the `obs_seqence_list` 
+to the text file containing the names of the ``obs_seq*.final``
+files associated with the tutorial.  Next, specify how the
+observations are displayed by defining the ``bin`` settings, which
+for this tutorial are set such that every day of observations
+are displayed individually. Because the tutorial is a global run
+we define the ``lonlim`` and ``latlim`` setting to include the
+entire globe.  For more information about the  ``obs_diag`` namelist
+settings go `here.  <https://docs.dart.ucar.edu/en/latest/assimilation_code/programs/obs_diag/threed_sphere/obs_diag.html>`__  
+
+::
+
+ &obs_diag_nml
+   obs_sequence_name = ''
+   obs_sequence_list = 'obs_seq_files_tutorial.txt'
+   first_bin_center =  2011, 1,  1, 0, 0, 0
+   last_bin_center  =  2011, 1,  5, 0, 0, 0
+   bin_separation   =     0, 0,  1, 0, 0, 0
+   bin_width        =     0, 0,  1, 0, 0, 0
+   time_to_skip     =     0, 0,  0, 0, 0, 0
+   max_num_bins     = 1000
+   trusted_obs      = 'null'
+   Nregions   = 1
+   lonlim1    =     0.0,     
+   lonlim2    =   360.0,   
+   latlim1    =   -90.0,     
+   latlim2    =    90.0,   
+   reg_names  = 'Globe', 
+   hlevel_edges =  0.0, 1.0, 2.0, 5.0, 10.0, 40.0
+   print_mismatched_locs = .false.
+   create_rank_histogram = .true.
+   outliers_in_histogram = .true.
+   use_zero_error_obs    = .false.
+   verbose               = .true.
+   /
+
+Next convert the information in the ``obs_seq*.final`` files into
+a netcdf format (``obs_diag_output.nc``) by executing the 
+``obs_diag`` executable.
+
+::
+ 
+ >  ./obs_diag
+
+Next, use Matlab to create the ``plot_rmse_xxx_evolution.m`` figures.
+Note that this function automatically plots the RMSE, where the ``copy``
+and the ``obsname`` variable are customizable.
+
+::
+
+ > cd <dartroot>/diagnostics/matlab
+ > module load matlab
+ >> matlab -nodesktop
+ >> fname   = '<dartroot>/models/clm/work/obs_diag_output.nc';
+ >> copy    = 'totalspread';
+ >> obsname = 'MODIS_LEAF_AREA_INDEX';
+ >> plotdat = plot_rmse_xxx_evolution(fname, copy, 'obsname', obsname);
+
+
+Second, the ``link_obs.m`` provides several features, including a spatial
+map of the observation locations with color coded DART QC values.  This allows
+the user to identify observation acceptance as a function of sub-regions.
+
+Next edit both the ``&obs_seq_to_netcdf_nml`` and ``&schedule_nml`` namelist sections
+within ``input.nml``.
+
+For this tutorial we plot a list of  ``obs_seq<>.final`` files as shown below,
```

Comment:
what is the `<>` for in file name?

Reply:
Switched to * notation which I used prior to this line for consistency.